std/net/curl.d

```diff
@@ -417,7 +417,7 @@ if (isCurlConn!Conn)
     }
 }
 
-@system unittest
+version(none) @system unittest
```

Comment:
Looks like you included these by accident?

Reply:
Ehm... My box has no connectivity so indeed.

Reply:
Fixed